docs/nodes/nodes-library/nodes/Sendy/README.md

```diff
@@ -0,0 +1,77 @@
+---
+permalink: /nodes/n8n-nodes-base.sendy
+description: Learn how to use the Sendy node in n8n
+---
+
+# Sendy
+
+[Sendy](https://sendy.co) is a self-hosted email newsletter application that lets you send trackable emails via AWS SES.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Sendy/README.md).
+:::
+
+## Basic Operations
+
+::: details Campaign
+- Create a campaign
+:::
+
+::: details Subscriber
+- Add a subscriber to a list
+- Count subscribers
+- Delete a subscriber from a list
+- Unsubscribe a user from a list
+- Get the status of a subscriber
+:::
+
+## Example Usage
+
+This workflow allows you to add a subscriber to a list and create and send a campaign using the Sendy node. You can also find the [workflow](https://n8n.io/workflows/727) on n8n.io. This example usage workflow uses the following nodes.
+- [Start](../../core-nodes/Start/README.md)
+- [Sendy]()
+
+The final workflow should look like the following image.
+
+![A workflow with the Sendy node](./workflow.png)
+
+### 1. Start node
+
+The Start node exists by default when you create a new workflow.
+
+
+### 2. Sendy node (add: subscriber)
+
+This node will add a subscriber to a list in Sendy.
+
+1. First of all, you'll have to enter credentials for the Sendy node. You can find out how to do that [here](../../../credentials/Sendy/README.md).
+2. Enter the subscriber's email address in the ***Email*** field.
+3. Enter the list id in the ***List ID*** field.
```

Comment:
Is this a prerequisite? If yes, it should be mentioned above. Also, how can one obtain a list ID?